File: LinkedList/SingleNode.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package linkedlist;

/**
 *
 * @author Kakatuahitam
 */
class SingleNode<Type> {
    Type data;
    SingleNode<Type> next;
    
    SingleNode(Type data){
        this(data, null);
    }
    
    SingleNode(Type data, SingleNode<Type> next){
        this.data = data;
        this.next = next;
    }
}

File: LinkedList/SingleNode.py
# -*- coding: utf-8 -*-
"""
Created on Tue Jun  2 04:11:55 2020

@author: Kakatuahitam
"""


class SingleNode:
    def __init__(self, data = None, next = None):
        self.data = data
        self.next = next
        
    def __repr__(self):
        return repr(self.data)